doc/examples/particle_filtering_fiber_tracking.py

```diff
@@ -0,0 +1,137 @@
+"""
+=================================================
+Particle Filtering Tractography
+=================================================
+Particle Filtering Tractography (PFT) [Girard2014]_ uses tissue partial
+volume estimation (PVE) to reconstruct trajectories connecting the gray matter,
+and not incorrectly stopping in the white matter or in the corticospinal fluid.
+It relies on a tissue classifier that identifies the tissue where the streamline
+stopped. If the streamline correctly stopped in the gray matter, the trajectory
+is kept. If the streamline incorrecly stopped in the white matter or in the
+corticospinal fluid, PFT uses anatomical information to find an alternative
+streamline segment to extend the trajectory. When this segment is found,
+the tractography continues until the streamline correctly stops in the gray
+matter.
+
+PFT finds an alternative streamline segment whenever the tissue classifier
+returns a position classified as 'INVALIDPOINT'.
+
+This example is an extension of the
+:ref:`probabilistic_fiber_tracking` example. We begin by loading the
+data, fitting a Constrained Spherical Deconvolution (CSD) reconstruction
+model and creating the probabilistic direction getter.
+"""
+
+import numpy as np
+
+from dipy.data import (read_stanford_labels, default_sphere,
+                       read_stanford_pve_maps)
+from dipy.direction import ProbabilisticDirectionGetter
+from dipy.io.trackvis import save_trk
+from dipy.reconst.csdeconv import (ConstrainedSphericalDeconvModel,
+                                   auto_response)
+from dipy.tracking.local import LocalTracking, ParticleFilteringTracking
+from dipy.tracking import utils
+from dipy.viz import fvtk
```

Comment:
`fvtk` will be deprecated, can you use the new viz API : 

`from dipy.viz import window, actor`